charts/microgateway/README.md.gotmpl

```diff
@@ -226,431 +231,372 @@ In case that the [Simple DSL configuration](#simple-dsl-configuration) does not
+        MIIEpAIBAAKCAQEAtXkTjHtDtutxyo1R6N4Eh18IzxoagAHPRzsdB5yeadcVr/bV
+        [...]
+        pGKu7aodwB4cD5YnfXTvUcTv5tNU0llRLG1J0bg1n9cCo0nTC9sUZw==
+        -----END RSA PRIVATE KEY-----
+  ```
+
+##### Route Reencrypt configuration
+With the TLS termination type "Reencrypt", HTTPS is terminated on the Openshift Router and re-encrypted to the Microgateway service.
+Because the default in the Microgateway Helm chart for `route.targetPort` is `https`, traffic inside the Openshift cluster is by default encrypted.
+The difference to TLS termination type "Edge" is:
+* The Microgateway certificate is validated against `route.tls.destinationCACertificate`.
+* It is enforced, that `route.targetPort=https` (Edge would also work with `route.targetPort=http`)
+
+In other words, the entire path of the connection is encrypted and verified, also within the Openshift cluster.
+
+  To setup Reencrypt TLS termination, use the following configuraion:
+  ```
+  route:
     enabled: true
-    termination: passthrough
-```
+    hosts:
+      - virtinc.com
+    tls:
+      enabled: true
+      certificate: |
+        -----BEGIN CERTIFICATE-----
+        MIIDizCCAnOgAwIBAgIJAMQE1QewYs4QMA0GCSqGSIb3DQEBCwUAMFwxCzAJBgNV
+        [...]
+        77RRptcoQJPvw50z9rJ4wkrb58raUKOqxg4Jn=
+        -----END CERTIFICATE-----
+      key: |
+        -----BEGIN RSA PRIVATE KEY-----
+        MIIEpAIBAAKCAQEAtXkTjHtDtutxyo1R6N4Eh18IzxoagAHPRzsdB5yeadcVr/bV
+        [...]
+        pGKu7aodwB4cD5YnfXTvUcTv5tNU0llRLG1J0bg1n9cCo0nTC9sUZw==
+        -----END RSA PRIVATE KEY-----
+      destinationCACertificate: | 
+        -----BEGIN CERTIFICATE-----
+        MIIDrIXdz54xcilsKUoepQkn9e0bmIUVuiXWcQrr8iqjYC+hINNmiq+4YX4lWq2M
+        [...]
+        K0RRA/rDxZnkbvtTd+hkoMu3Or+pqpOrp2n1pbtzoVl9Hg==
+        -----END CERTIFICATE-----
+  ```
 
-## Environment variables
-With the Helm chart environment variables can be set.   
-These can then be used in the DSL configuration.   
-Below is an example of how to set environment variables and then use them in the DSL.   
-
-env-variables.yaml
-```
-config:
-  global:
-    env:
-      - name: WAF_CFG_OPERATIONALMODE
-        value: production
-      - name: WAF_CFG_LOGONLY
-        value: false
-```
-
-custom-values.yaml
-```
-config:
-  simple: 
-    mapping:
-      operationalMode: "@@WAF_CFG_OPERATIONALMODE@@"
-      denyRules:
-        logOnly: "@@WAF_CFG_LOGONLY@@"
-```
+##### Route Passthrough configuration
+With the TLS termination type "Passthrough", HTTPS traffic is sent directly to the Microgateway, without decrypting it on the Route.
+Therefore, no certificates need to be configured on the Route and termination takes place in the Microgateway.
 
-## Probes
-In Kubernetes and Openshift probes are used to determine if a pod is ready to process requests or not.  
+  To setup Passthrough TLS termination, use the following configuraion:
+  ```
+  route:
+    enabled: true
+    hosts:
+      - virtinc.com
```

Comment:
Path: ""
Its not possible to configure Passthrough with a Path.

Reply:
Done.